### PR TITLE
Add border to each query in time series chart editor

### PR DESCRIPTION
**Pull request by cndonovan**
Now queries have nice borders that match the designs!

Screen Shot 2022-10-20 at 12 21 01 PM

Also, updated the names of the handler functions to be more consistent in this file (Steven lemme know what you think)

### CODE REVIEW

**Review by Nexucis — APPROVED** [review]
so shiny 🤩. Thanks for the changes @cndonovan !!

File: ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx [hunk 1]
```diff
@@ -103,8 +103,8 @@ export function TimeSeriesChartOptionsEditor(props: TimeSeriesChartOptionsEditor
         Add Query
       </Button>
       {queries.map((query, i) => (
-        <Stack key={i}>
-          <Stack direction="row">
+        <Stack key={i} spacing={1}>
+          <Stack direction="row" borderBottom={1} borderColor="grey.300">
```
Comment (Nexucis) [inline ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx:107]: did you check, it's visible in the dark mode ? :)

Comment (cndonovan) [reply ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx:107]: Ya, the grey.x colors are all safe, they are mapped to the right value in each mode

Comment (cndonovan) [reply ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx:107]: Screen Shot 2022-10-20 at 3 19 29 PM

**Comment by cndonovan** [comment]
Feel free to merge this if you want, I think it's pretty safe (but also feel free to wait for a UI engineer review if you want to)

File: ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx (hunk 1 above)
Comment (Nexucis) [reply ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx:107]: perfect !

**Pull request merged**